lib/chef/http/authenticator.rb

```diff
@@ -123,9 +123,18 @@ def self.detect_certificate_key(client_name)
         end
       end
 
+      def self.get_cert_user
+        Chef::Config[:auth_key_registry_type] == "user" ? store = "CurrentUser" : store = "LocalMachine"
+      end
+
+      def self.get_registry_user
+        Chef::Config[:auth_key_registry_type] == "user" ? store = "HKEY_CURRENT_USER" : store = "HKEY_LOCAL_MACHINE"
+      end
+
```

Comment:
This shouldn't do `store == ...` in the two cases, just return the value in those cases...

Reply:
done